apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/attach-focused-launch.tsx

```diff
@@ -44,10 +44,7 @@ registerPlugin( 'a8c-editor-editor-focused-launch', {
 
 		// '?should_launch' query arg is used when the mandatory checkout step
 		// is redirecting to an external payment processing page (eg: Paypal)
-		const shouldLaunch = hasQueryArg(
-			inIframe() ? getCurrentLaunchFlowUrl() : window.location.href,
-			IMMEDIATE_LAUNCH_QUERY_ARG
-		);
+		const shouldLaunch = hasQueryArg( getCurrentLaunchFlowUrl(), IMMEDIATE_LAUNCH_QUERY_ARG );
```

Comment:
The `inIframe()` check was redundant, since `getCurrentLaunchFlowUrl()` already uses `window.location.href` when in Wp Admin